mubench.pipeline/tasks/implementations/check.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import List
+
+from tasks.project_task import Requirement
+from tasks.project_task import ProjectTask
+from utils.shell import Shell
+
+
+class Check(ProjectTask):
+    def __init__(self):
+        super().__init__()
+
+    def start(self):
+        self._check_all_requirements()
+
+    def process_project(self, *_):
+        return self.ok()
+
+    def _check_all_requirements(self):
+        logger = logging.getLogger("requirements")
+        logger.info("Checking all requirements...")
```

Comment:
I'm fine with removing the "Checking requirements" output from the runner. I agree that this is non of its business no more. I was referring to the "Starting check at..." line. The "check" part comes from the task name. By renaming the task it would become "Starting requirementscheck at...", which basically duplicates the subsequent line (modulo the unfortunate formatting, which I plan to improve by splitting the task name at camel case in the future, which would make this "Starting requirements check at ...").